### PR TITLE
Fix #139: Add "from" field to chat events

### LINKED ISSUE #139: "from" field is missing in GroupEventMessage
When a bot is added to a group, a message like this is sent by the API:

```json5
{
	"update_id": 845170060,
	"message": {
		"message_id": 196,
		"from": {
			"id": 123, // A user, who added a bot to a group
			"is_bot": false,
			"first_name": "Name",
			"username": "username",
			"language_code": "en"
		},
		"chat": {
			"id": 42,
			"title": "Group",
			"type": "group",
			"all_members_are_administrators": true
		},
		"date": 1601159770,
		"new_chat_participant": {
			"id": 456, // Bot ID
			"is_bot": true,
			"first_name": "Bot",
			"username": "BotBot"
		},
		"new_chat_member": {
			"id": 456,
			"is_bot": true,
			"first_name": "Bot",
			"username": "BotBot"
		},
		"new_chat_members": [
			{
				"id": 456,
				"is_bot": true,
				"first_name": "Bot",
				"username": "BotBot"
			}
		]
	}
}
```

It is parsed as `MessageUpdate`:

```
MessageUpdate(
    updateId=845170060,
    data=GroupEventMessage(
        messageId=196,
        chat=GroupChatImpl(
            id=ChatId(chatId=42),
            title=Group
        ),
        chatEvent=NewChatMembers(
            members=[
                CommonBot(
                    id=ChatId(chatId=456),
                    username=Username(username=@BotBot),
                    firstName=Bot,
                    lastName=
                )
            ]
        ),
        date=Sat, 26 Sep 2020 22:36:10 UTC
    )
)
```

As you see, the information about the user is lost. Is it possible to retrieve it somehow?

### PR DESCRIPTION
**Pull request by madhead**

### CODE REVIEW

**Comment by madhead** [comment]
I have no idea why those GitHub checks failed…

**Comment by InsanusMokrassar** [comment]
Checks may be failed due to some settings in your account:) don't worry about it - here the most important that Travis build passed

**Comment by InsanusMokrassar** [comment]
@madhead It is good idea to add the source of event, but I am not sure when `from` field is filled in the events. It would be glad if you will check that and include here results. I plan to do that asap, but unfortumatelly today I will not be able for it

**Comment by InsanusMokrassar** [comment]
According to the documentation ( https://core.telegram.org/bots/api#message ) only channel messages (including events) have no `from` field

**Comment by madhead** [comment]
@InsanusMokrassar, yes, you are correct. Channel messages seem to have no `from`, so group and supergroup events should have it. That's why I added the field only to those two corresponding clasees.

**Comment by InsanusMokrassar** [comment]
I think, it would be better to create `GroupEventMessage : ChatEventMessage, FromUserMessage` and `SupergroupEventMessage`, and change parent interfaces for the target classes with renaming (`GroupEventMessage` -> `CommonGroupEventMessage`, for example)

**Comment by madhead** [comment]
@InsanusMokrassar, check the latest changes, please. Is it what you mean?

File: TelegramBotAPI-core/src/commonMain/kotlin/com/github/insanusmokrassar/TelegramBotAPI/types/message/CommonSupergroupEventMessage.kt [hunk 1]
```diff
@@ -1,14 +1,18 @@
...
 
-data class SupergroupEventMessage(
+data class CommonSupergroupEventMessage(
     override val messageId: MessageIdentifier,
+    override val user: User,
     override val chat: SupergroupChat,
     override val chatEvent: SupergroupEvent,
     override val date: DateTime
-) : ChatEventMessage
+) : GroupEventMessage
```
Comment (InsanusMokrassar) [inline TelegramBotAPI-core/src/commonMain/kotlin/com/github/insanusmokrassar/TelegramBotAPI/types/message/CommonSupergroupEventMessage.kt:18]: Here better to use `SupergroupEventMessage`. It must extends `GroupEventMessage`

**Review by InsanusMokrassar — CHANGES_REQUESTED** [review]

**Comment by madhead** [comment]
@InsanusMokrassar, done

File: TelegramBotAPI-core/src/commonMain/kotlin/com/github/insanusmokrassar/TelegramBotAPI/types/message/CommonSupergroupEventMessage.kt (hunk 1 above)
Comment (madhead) [reply TelegramBotAPI-core/src/commonMain/kotlin/com/github/insanusmokrassar/TelegramBotAPI/types/message/CommonSupergroupEventMessage.kt]: Done

**Review by InsanusMokrassar — APPROVED** [review]

**Pull request merged**